src/Core/OrganizationFeatures/OrganizationServiceCollectionExtensions.cs

```diff
@@ -171,6 +176,14 @@ private static void AddOrganizationUserCommandsQueries(this IServiceCollection s
         services.AddScoped<IAuthorizationHandler, OrganizationUserUserMiniDetailsAuthorizationHandler>();
         services.AddScoped<IAuthorizationHandler, OrganizationUserUserDetailsAuthorizationHandler>();
         services.AddScoped<IHasConfirmedOwnersExceptQuery, HasConfirmedOwnersExceptQuery>();
+
+        services.AddScoped<IInviteOrganizationUsersCommand, InviteOrganizationUsersCommand>();
+        services.AddScoped<ISendOrganizationInvitesCommand, SendOrganizationInvitesCommand>();
+
+        services.AddScoped<IInviteUsersValidator, InviteUsersValidator>();
+        services.AddScoped<IInviteUserOrganizationValidator, InviteUserOrganizationValidator>();
+        services.AddScoped<IPasswordManagerInviteUserValidator, PasswordManagerInviteUserValidator>();
+        services.AddScoped<IEnvironmentValidator, EnvironmentValidator>();
```

Comment:
Suggested naming now that these are in global DI - it's particularly important that they're clearly namespaced:

- `IInviteUsersValidator`
- `IInviteUsersOrganizationValidator`
- `IInviteUsersPasswordManagerValidator`
- `IInviteUsersEnvironmentValidator`